app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -42,7 +42,7 @@ def pvm_instances
         :flavor            => persister.flavors.lazy_find(instance.sys_type),
         :location          => _("unknown"),
         :name              => instance.server_name,
-        :vendor            => "ibm",
+        :vendor            => "ibm_cloud",
```

Comment:
If we're going to change the vendor of existing vm records we're going to need a schema data migration to change existing db records that have `:vendor => "ibm"`

Reply:
Good catch. I'll work on a patch today.

Reply:
https://github.com/ManageIQ/manageiq-schema/pull/587